internal/database/filter.go

```diff
@@ -70,7 +70,7 @@ func (r *FilterRepo) find(ctx context.Context, tx *Tx, params domain.FilterQuery
 		From("filter f")
 
 	if params.Search != "" {
-		queryBuilder = queryBuilder.Where("f.name LIKE ?", fmt.Sprint("%", params.Search, "%"))
+		queryBuilder = queryBuilder.Where(sq.Like{"f.name": params.Search + "%"})
```

Comment:
Is it intentional to skip the prefix `%`?

Reply:
Yeah. Right now if you search for `Dog` you'll get Dog.S02E04 and `Mary.Poppins.S01E04.Clifford.The.Dog`.